PKGBUILD

```diff
@@ -27,11 +27,11 @@ optdepends=('pepper-flash: support for Flash content'
 install=inox.install
 source=(https://commondatastorage.googleapis.com/chromium-browser-official/chromium-$pkgver.tar.xz
         chromium-launcher-$_launcher_ver.tar.gz::https://github.com/foutrelis/chromium-launcher/archive/v$_launcher_ver.tar.gz
-        https://raw.githubusercontent.com/gcarq/inox-patchset/$pkgver/inox.desktop
```

Comment:
Is there a reason why you removed the inox.desktop file? I know it's not the best solution but having the desktop file only in the AUR git repo is a little bit cumbersome

Reply:
We don't need to include it anymore since Chromium provides a template file for it at `chrome/installer/linux/common/desktop.template`, which is proccessed at https://github.com/gcarq/inox-patchset/blob/3fec87bc60ac5d3912961bfaee30a58a568a672e/PKGBUILD#L318

This change comes from the official PKGBUILD and I probably should have mentioned it in the description, sorry.

Reply:
I see. Thanks :-)